### PR TITLE
Add Blue Style Guide badge to README

### PR DESCRIPTION
**Pull request by Zapiano**
After the team agreed to follow Blue Style Guide, the Blue Style Guide badge is being added to the REAMDE file together with a paragraph informing that we are starting to move in this direction.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,11 +16,16 @@ returns on investment. ADRIA uses a set of dynamic Multi-Criteria Decision Analy
...
-Setup and usage is demonstrated in the [documentation](https://open-aims.github.io/ADRIA.jl/stable/usage/getting_started/)
+Setup and usage is demonstrated in the
+[Documentation](https://open-aims.github.io/ADRIA.jl/stable/usage/getting_started/).
+
+For developers, refer to the
+[Developers setup guide](https://open-aims.github.io/ADRIA.jl/stable/development/development_setup/).
 
-For developers, refer to the [Developers setup guide](https://open-aims.github.io/ADRIA.jl/stable/development/development_setup/)
+This project is currently moving to use [Blue Style Guide](https://github.com/invenia/BlueStyle).
+Please, try to follow the style guide when submiting a PR.
```
Comment (ConnectedSystems) [inline README.md:31]: Move these statements above or next to the badge.

Remove "please" and "try".
https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html#eliminate-needless-politeness
https://developers.google.com/style/word-list#please

**Review by ConnectedSystems — CHANGES_REQUESTED** [review]
Thanks Pedro, just minor change to style.

File: README.md (hunk 1 above)
Comment (Zapiano) [reply README.md]: @ConnectedSystems done. Just one note: moving these statements above the badge make them the first thing you read when you enter ADRIA's README. That's why I had put them at the end.

Comment (ConnectedSystems) [reply README.md]: Okay, below? Currently the statement is too far from the badge so it's a little deceptive. Alternatively, move the badge down until at least most of the code has been updated.

Comment (Zapiano) [reply README.md]: When I try to put them at the same line using Markdown only, it get's vertically unaligned

image

And when I jump one line between them, the distance between the badge and the next paragraph is bigger than to the previous:

image


For that reason I have used HTML.

Comment (ConnectedSystems) [reply README.md]: Yep, that's fine.

**Comment by ConnectedSystems** [comment]
Looks good now, thanks.

You don't need to force push changes that incorporate feedback by the way.

**Review by ConnectedSystems — APPROVED** [review]

**Pull request merged**